tokenizers/src/lib.rs

```diff
@@ -26,11 +26,13 @@
 //! use tokenizers::tokenizer::{Result, Tokenizer};
 //!
 //! fn main() -> Result<()> {
-//!     let tokenizer = Tokenizer::from_pretrained("bert-base-cased", None)?;
-//!
-//!     let encoding = tokenizer.encode("Hey there!", false)?;
-//!     println!("{:?}", encoding.get_tokens());
+//!     #[cfg(feature = "http")]
+//!     {
+//!         let tokenizer = Tokenizer::from_pretrained("bert-base-cased", None)?;
 //!
+//!         let encoding = tokenizer.encode("Hey there!", false)?;
+//!         println!("{:?}", encoding.get_tokens());
+//!     }
```

Comment:
Do you mind hiding these as they are mostly necessary for the test but shouldn't be displayed in the doc (since it's a default feature)

```suggestion
//!     # #[cfg(feature = "http")]
//!     # {
//!      let tokenizer = Tokenizer::from_pretrained("bert-base-cased", None)?;
//!
//!      let encoding = tokenizer.encode("Hey there!", false)?;
//!      println!("{:?}", encoding.get_tokens());
//!     # }
```

Reply:
You will need to modify the README.md file too (`cargo readme > README.md` should be enouh)

Reply:
Wow I spent so long trying to figure out how to do that, nice, fixed!